shims/aggregator/pom.xml

```diff
@@ -43,6 +43,17 @@
                 </dependency>
             </dependencies>
         </profile>
+        <profile>
+            <id>databricks310</id>
+            <dependencies>
+                <dependency>
+                    <groupId>com.nvidia</groupId>
+                    <artifactId>rapids-4-spark-shims-spark310-databricks_${scala.binary.version}</artifactId>
+                    <version>${project.version}</version>
+                    <scope>compile</scope>
+                </dependency>
+            </dependencies>
+        </profile>
         <profile>
             <!-- use a separate profile to just pull databricks from maven repository without building it -->
             <id>include-databricks</id>
```

Comment:
yeah if its alright I'll file a followup for this because I need to do the CI/CD for databricks 8.2 and at that point was thinking of removing some individual ones again.  I added it to https://github.com/NVIDIA/spark-rapids/issues/2390